packages/orchestrator-ui-components/src/components/WfoForms/formFields/deprecated/VlanField.tsx

```diff
@@ -156,16 +156,10 @@ function Vlan({
         .join(',');
 
     useEffect(() => {
-        if (subscriptionId && isFetched && !portIsTagged && value !== '0') {
-            onChange('0');
-        } else if (
-            !disabled &&
-            ((!subscriptionId && value !== '') ||
-                (subscriptionId && portIsTagged && value === '0'))
-        ) {
+        if (!initialValue) {
             onChange('');
         }
-    }, [onChange, subscriptionId, value, isFetched, portIsTagged]);
+    }, [initialValue, onChange, subscriptionId, isFetched]);
```

Comment:
Could you describe in a comment what this fix does and where to watch out for if we ever want to touch this component in the future?